modules/installation-ibm-z-troubleshooting-and-debugging.adoc

```diff
@@ -7,7 +7,7 @@
 = Collecting debugging information
 
 You can gather debugging information that might help you to troubleshoot and
-debug certain issues with an {product-title} installation on IBM Z.
+debug certain issues with an {product-title} installation on {ibmzProductName}.
```

Comment:
I could not find this module

Reply:
Another nice catch. No modules include this file.

Reply:
Good catch. Since 4.10 we don't need that anymore but it's still live in 4.9: https://docs.openshift.com/container-platform/4.9/installing/installing_ibm_z/installing-ibm-z.html#installation-ibm-z-troubleshooting-and-debugging_installing-ibm-z
@rolfedh @EricPonvelle How would you proceed? Keep it in the PR and I make a reminder to remove the file once 4.9 goes out of service. Or add a comment in the file?

Reply:
Keeping it in the PR makes sense to me. Let's mark this conversation as resolved.